enterprise/apps.py

```diff
@@ -17,7 +17,7 @@ class EnterpriseConfig(AppConfig):
 
     name = "enterprise"
     valid_extensions = [".png", ]
-    image_size = 250 * 1024
+    image_size = 4 * 1024
```

Comment:
It seems me, logo should be enough to have 4kb in size ( not need to change frequently). However we need to scale that logo on client side that will be happened on my other [PR](https://github.com/edx/edx-platform/pull/14064)